src/config.books.json

```diff
@@ -157,7 +158,8 @@
   },
   "afe4332a-c97f-4fc4-be27-4e4d384a32d8": {
     "defaultVersion": "18.2",
-    "archiveOverride": "/apps/archive/20210713.205645"
+    "archiveOverride": "/apps/archive/20210713.205645",
```

Comment:
is there any value in keeping the archiveOverride if the book is retired? it'll never be used right?

Reply:
no, if we're on dev or heroku we should be able to load any archive content, production has `UNLIMITED_CONTENT=false` meaning you can only see content that is specified in `config.books.json`